visualization_marker_tutorials/CHANGELOG.rst

```diff
@@ -2,6 +2,10 @@
 Changelog for package visualization_marker_tutorials
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+0.11.0 (2019-04-05)
```

Comment:
This version won't work, as it prevents us from continuing to progress on the ROS 1 branch.

Also, this is not how releases work... Please just have here the porting changes, once that's merged we can look at a release, documented here: https://index.ros.org/doc/ros2/Tutorials/Releasing-a-ROS-2-package-with-bloom/